### PR TITLE
JSP: update to U15.0 

### PR DESCRIPTION
**Pull request by srl295**
For: #271 
Fixes: #165 

- update the UpdateJspFiles tool
- run UpdateJspFiles to update for U15
- update docs concerning running jetty locally

### CODE REVIEW

File: unicodetools/src/main/java/org/unicode/tools/UpdateJspFiles.java [hunk 1]
```diff
@@ -140,7 +141,9 @@ private static void generateSubtagNames() throws IOException {
 
     private static void copyOtherProps(VersionInfo fromVersion) throws IOException {
         copyTextFiles(
-                UNICODE_TOOLS_DIR.resolve("src/main/resources/org/unicode/props"), // TODO: will break with mavenize
+                UNICODE_TOOLS_DIR.resolve("src/main/resources/org/unicode/props"), // TODO: will
```
Comment (markusicu) [inline unicodetools/src/main/java/org/unicode/tools/UpdateJspFiles.java:144]: is the TODO still relevant?

File: unicodetools/src/main/java/org/unicode/tools/UpdateJspFiles.java [hunk 2]
```diff
@@ -140,7 +141,9 @@ private static void generateSubtagNames() throws IOException {
 
     private static void copyOtherProps(VersionInfo fromVersion) throws IOException {
         copyTextFiles(
-                UNICODE_TOOLS_DIR.resolve("org/unicode/props"), // TODO: will break with mavenize
+                UNICODE_TOOLS_DIR.resolve("src/main/resources/org/unicode/props"), // TODO: will
+                // break with
+                // mavenize
```
Comment (srl295) [inline unicodetools/src/main/java/org/unicode/tools/UpdateJspFiles.java:146]:
```suggestion
                UNICODE_TOOLS_DIR.resolve("src/main/resources/org/unicode/props"),
```

File: unicodetools/src/main/java/org/unicode/tools/UpdateJspFiles.java (hunk 1 above)
Comment (srl295) [reply unicodetools/src/main/java/org/unicode/tools/UpdateJspFiles.java]: good point, it's now fixed

**Review by markusicu — APPROVED** [review]
lgtm tnx

**Pull request merged**